faker/providers/misc/__init__.py

```diff
@@ -91,7 +91,7 @@ def binary(cls, length=(1 * 1024 * 1024)):
 
         Default blob size is 1 Mb.
         """
-        return os.urandom(length)
```

Comment:
`int` didn't gain a `to_bytes` method until Python 3, and this project still supports Python 2.7.